op-chain-ops/cmd/celo-migrate/ancients.go

```diff
@@ -104,56 +94,67 @@ func migrateAncientsDb(ctx context.Context, oldDBPath, newDBPath string, batchSi
 
 func readAncientBlocks(ctx context.Context, freezer *rawdb.Freezer, startBlock, endBlock, batchSize uint64, out chan<- RLPBlockRange) error {
 	defer close(out)
-
 	for i := startBlock; i < endBlock; i += batchSize {
-		count := min(batchSize, endBlock-i+1)
+		count := min(batchSize, endBlock-i)
```

Comment:
The audit missed this indexing error. It should be endBlock - i because endBlock is not inclusive. This didn't cause any problems before because the freezer.AncientRange function returns at most count elements, and this indexing error could only cause a problem when reading the last section of the ancients (when the number of remaining blocks is less than batchSize)